config/controller.yaml

```diff
@@ -78,7 +78,7 @@ spec:
           # The shell image must be root in order to create directories and copy files to PVCs.
           # gcr.io/distroless/base:debug as of October 21, 2021
```

Comment:
Can you update this comment?

Reply:
Yes. Please see https://github.com/tektoncd/pipeline/pull/4588/commits/8d88b18b39fa468912b5b98ac1fbc3bfa1b31b1b. Thank you.